ResourceWatch/Script_geostore_IDs_OW-79.ipynb

```diff
@@ -0,0 +1,1161 @@
+       "2      Nigeria      NGA   5.607847   5.174516       8474   20404.0  \n",
+       "3       Angola      AGO  12.684313 -11.326831       8478   34318.0  \n",
+       "4     Tanzania      TZA  39.926449  -7.641209       8479   16008.0  "
+      ]
+     },
+     "execution_count": 28,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "eez_table = 'com_011_1_maritime_boundaries_territorial_waters'\n",
+    "eez_df = cf.io.carto.read_carto(eez_table, credentials=creds)\n",
+    "\n",
+    "eez_df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Creating geojson and getting geostore id"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Countries geometries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 62,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 413\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 413\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 500\n",
+      "Failed to register geostore. Error 502\n"
+     ]
+    }
+   ],
+   "source": [
+    "countries_g_id_dict = {}\n",
```

Comment:
why not generalizing this as a function? also for loops can be slow you can in the future think of ussing eval or apply methods that apply a function over all rows

Also  if you do this:
```python 
for i in countries_df.index:
    g = countries_df.loc[[i]].the_geom.to_json()
```

You will not need to create a custom function to get the geometry as a geojson